opencodelists/tests/factories.py

```diff
@@ -14,7 +14,14 @@ class Meta:
     @classmethod
     def _create(cls, model_class, *args, **kwargs):
         """Override the default ``_create`` with our actions."""
-        return actions.create_organisation(*args, **kwargs)
+        organisation = actions.create_organisation(*args, **kwargs)
+        organisation.regular_user = UserFactory()
```

Comment:
I'm not sure I understand this line.  There's no `regular_user` field on the `Organisation` model, is this to make it easy to grab an authorised User from the Organisation instances inside tests?

Reply:
Comment added in 76d9db284e32c47345a2cb287b6a8365f67ad9d0.